doc/related_projects.rst

```diff
@@ -67,11 +67,13 @@ enhance the functionality of scikit-learn's estimators.
 - `mlxtend <https://github.com/rasbt/mlxtend>`_ Includes model visualization
   utilities.
 
+- `Fast svmlight / libsvm file loader <https://github.com/mblondel/svmlight-loader>`_
+  Fast and memory-efficient svmlight / libsvm file loader for Python.
```

Comment:
Maybe we can put it under "Data formats"? It's just a file loader, not really related to "inspection and visualisation".

Reply:
Done.